MANIFEST.in

```diff
@@ -1,15 +1,14 @@
-include LICENSE
-include README.rst
+# Note: See the PyPA documentation for a list of file names that are included/excluded by default:
+# https://packaging.python.org/guides/using-manifest-in/#how-files-are-included-in-an-sdist
+# Please only add entries here for files that are *not* already handled by default.
+
 include CHANGELOG.rst
 include CONTRIBUTING.md
-include tox.ini
 include Makefile
-recursive-include tests *.py
-recursive-include tests *.h
-recursive-include tests *.m
-recursive-include docs *.bat
-recursive-include docs *.png
-recursive-include docs *.py
-recursive-include docs *.rst
-recursive-include docs *.txt
-recursive-include docs Makefile
+include tox.ini
+include docs/Makefile
+include docs/make.bat
+include docs/requirements_docs.txt
+include docs/spelling_wordlist
+recursive-include docs *.png *.py *.rst
+recursive-include tests *.h *.m *.py
```

Comment:
This looks mostly right, but `check-manifest` pointed out to me that `changes/template.rst` isn't picked up by this configuration. It also seems to be complaining about the docs/_build folder, but I can't work out why...

Reply:
That's a tool I didn't know about, thanks for the tip!

You're completely right, changes/template.rst is missing, I'll fix that.

I'm not sure why so much from the docs/_build directory is being included... The .png files make sense (because we `recursive-include docs *.png`), but the generated .rst.txt *shouldn't* match any of the patterns... In any case, it might be safest to explicitly `prune docs/_build`, because nothing from that directory should appear in an sdist.

Reply:
Yeah - I can't explain docs/_build either; it *should* be excluded by the gitignore, but for some reason, check-manifest is still picking it up. Might be a bug with check-manifest...

Regardless, I agree pruning docs/_build makes sense.

Reply:
> it _should_ be excluded by the gitignore

`setup.py sdist` doesn't care about gitignore though :) I think check-manifest is correctly detecting that there's a problem here, but the message is misleading - in this case the proper fix is not to add the directory to VCS, but to exclude it from the sdist.

Reply:
🤦‍♂️ Of course. Makes perfect sense.